### DISCUSSION TITLE
Planning a route in another country: search radius

### DISCUSSION THREAD

**Discussion by what-aboot** [osmandapp/OsmAnd#20587] [Maps Related]
I've searched and can't find any posts related to search radius.

Search radius defaults to 5km. I am trying to plan a route through India.

Surely search radius should be in tune with the zoom level or scale of the map displayed?

I am in Scotland. 

I'm trying to plan a motorcycle route through North West India. I know place names I want to pass through, and I'm creating new routes for various sections. Once done, I will show various combinations "on map" to see which work, at which point I will delete any others.

When I type in the name of any place in India, some 4400km from where I am, the search radius appears to be 5km from where I presently am, returning no result, prompting me to **increase the search radius** by pressing a button. On pressing the search, it  repeats, at double the previous distance, ie 10km. No amount of pressing that button will reach 4400 km.

There is an option "SHOW ON MAP". I press this and move the displayed map to Ladakh where I'm looking for a place called Leh. I don't exactly know where it is (which is why I was searching for it in the first place). At this point I would expect a repeat "search" to be searching _the area I'm showing on the map_ **at the scale** I'm using on the map (ie a search radius of 5km doesn't really compute when I have the entire Himalayas showing on the map). 

Alas, returning to the search text entry box searches with 100km of my present position again, with no results.

**Comment by DmitryAlexei** [comment]
Search radius should work from the center of the map
Have you tried OsmAnd map for planning routes? https://osmand.net/map/
On the map you can create and download .gpx routes. With PRO subscription, .gpx synchronization can be done via OsmAnd cloud.
